app/src/main/java/com/dcrandroid/fragments/OverviewFragment.kt

```diff
@@ -106,13 +130,17 @@ class OverviewFragment : BaseFragment(), ViewTreeObserver.OnScrollChangedListene
         setToolbarTitle(R.string.overview, false)
         scrollView.viewTreeObserver.addOnScrollChangedListener(this)
 
+        isBalanceHidden = multiWallet!!.readBoolConfigValueForKey(Constants.SHOW_HIDE_BALANCE, false)
```

Comment:
We should not use `!!`

Reply:
the use of `!!` is important and acceptable for certain use cases, for example you need the app to crash when a vital package or resource isn't available

`multiwallet` is part of the foundation of our apps, if for some reason multiwallet tends to be nil, there are countless errors that would occur as a result of that, and for that scenario, it's best for the app to panic and shutdown